package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4336,155 +4524,24 @@ def load_new(self, filename, **kwargs):
         return filename, self.trajectory.format
 
     def select_atoms(self, sel, *othersel, **selgroups):
-        """Select atoms using a CHARMM selection string.
-
-        Returns an :class:`AtomGroup` with atoms sorted according to their
-        index in the psf (this is to ensure that there aren't any duplicates,
-        which can happen with complicated selections).
-
-        Existing :class:`AtomGroup` objects can be passed as named arguments,
-        which will then be available to the selection parser.
-
-        Subselections can be grouped with parentheses.
-
-        Example::
-           >>> sel = universe.select_atoms("segid DMPC and not ( name H* or name O* )")
-           >>> sel
-           <AtomGroup with 3420 atoms>
-
-           >>> universe.select_atoms("around 10 group notHO", notHO=sel)
-           <AtomGroup with 1250 atoms>
+        """Selection of atoms using the MDAnalysis selection syntax.
```

Comment:
So here it's "MDAnalysis selection syntax" while for `AtomGroup.select_atoms` it's a "CHARMM selection string." For consistency, choose one.